src/rendering/Fields.zig

```diff
@@ -398,8 +398,8 @@ test "enum literal " {
 }
 
 test "enum" {
-    const Options = enum { AreYouSure, Totally };
-    var data = .{ .value = Options.AreYouSure, .level = .{ .value = Options.Totally } };
+    const Options = enum { areyousure, totally };
```

Comment:
```suggestion
    const Options = enum { are_you_sure, totally };
```

Reply:
I force pushed the suggested changes. I hope that's ok.